api/api/serializers.py

```diff
@@ -57,7 +60,22 @@ def create(self, validated_data):
         lon = point.x
         stadsdeel = BagGeoSearchAPI().get_stadsdeel(lat, lon)
         validated_data['stadsdeel'] = stadsdeel
-        return super().create(validated_data)
+
+        rapport_document = validated_data.pop('rapport_document', None)
+        design_document = validated_data.pop('design_document', None)
+
+        # TODO implement file upload to objectstore
+
+        spot = super().create(validated_data)
+
+        if rapport_document:
+            Document.objects.create(type=Document.DocumentType.Rapportage,
+                                    spot=spot, filename='TODO')
+        if design_document:
+            Document.objects.create(type=Document.DocumentType.Ontwerp,
+                                    spot=spot, filename='TODO')
```

Comment:
TODO filename?

Reply:
Intended; there is another user story to actually implement the objectstore, and setting the proper filename.